include/revng/StackAnalysis/IndirectBranchInfoPrinterPass.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/IR/PassManager.h"
+
+#include "revng/BasicAnalyses/GeneratedCodeBasicInfo.h"
+#include "revng/Support/revng.h"
+
+class IndirectBranchInfoPrinterPass
+  : public llvm::PassInfoMixin<IndirectBranchInfoPrinterPass> {
+  GeneratedCodeBasicInfo *GCBI = nullptr;
```

Comment:
Again: passes should not hold any storage. Make an `Impl` class or make `serialize` a `static` function and pass GCBI to it.